application/src/main/java/org/togetherjava/tjbot/DatabaseListener.java

```diff
@@ -51,11 +90,13 @@ private void handlePutMessage(String message, MessageReceivedEvent event) {
         String key = data[1];
         String value = data[2];
 
+        // Save the value in the database
```

Comment:
That's probably not necessary either here? If you feel it is really needed I'd rather extract the try block in a method named .e.g `savePair`.